antlr_plsql/ast.py

```diff
@@ -43,13 +47,29 @@ def _from_sql_script(cls, visitor, ctx):
             if not isinstance(child, Tree.TerminalNodeImpl):
                 body.append(visitor.visit(child))
 
-        return cls(ctx, body = body)
+        return cls(ctx, body=body)
 
 
 class SelectStmt(AstNode):
-    _fields = ['pref', 'target_list', 'into_clause', 'from_clause', 'where_clause',
```

Comment:
Small, tangential opinion. I'm not a huge fan of converting these to big vertical lists, since 

1. AFAIK humans read faster from left-to-right
2. The wide form has a similar density to the SQL query it's describing

Reply:
Haven't been able to fully review yet, but I wanted to react on this! 

I don't agree!

My reasoning is:

1. Reading english is different from reading code. I generally find words faster if they're vertically aligned because you generally scan the first few letters first and this makes it easier!
2. Don't know about that. I typically write SQL more vertical than horizontal as well TBH :P. But I kinda get what you mean.

Also consider the following,

- Vertical lists are better for git diffing! Less unecessary diffed content.
- It's easier to restructure them (e.g. reorder, remove things, ...). Definitely with VIM.
- As I mentioned before, I'd argue it's actually quicker to read, but we should do an experiment on this. (Maybe there's already been experiments on this :-O)

Reply:
Also `black` reformats it this way so I generally just listen :-D

Reply:
It's black formatting, for easy consistency.